kuma/static/styles/components/payments/_page.scss

```diff
@@ -214,4 +196,145 @@
             border-bottom: 2px solid $text-color;
         }
     }
+
+
+    &.manage-subscriptions {
+        .active-subscriptions {
+            border: 2px solid $text-color;
+            padding: 20px;
+            margin-bottom: 10px;
+
+            p {
+                margin: 0;
+            }
+
+            ul {
+                display: flex;
+                flex-direction: column;
+                font-weight: bold;
+                margin: 0;
+
+                @media #{$mq-tablet-and-up} {
+                    flex-direction: row;
+                    li:not(:first-child) {
+                        margin-left: 10%;
+                    }
+                }
+            }
+
+            .label {
+                padding-left: 30px;
+                &::after {
+                    content: ': ';
+                }
+
+                &.amount {
+                    background: url('/static/img/subscriptions/money.svg') 0 50% no-repeat;
+                }
+
+                &.credit-card {
+                    background: url('/static/img/subscriptions/credit-card.svg')
+                        0 50% no-repeat;
+                }
+
+                @media #{$mq-tablet-and-up} {
+                    display: block;
+                    &::after {
+                        content: '';
+                    }
+                }
+            }
+        }
+
+        form {
+            background: $red-light;
+            border: 1px solid $brand-red;
+            clear: both;
+            padding: 20px;
+            overflow: hidden;
+
+            strong {
+                display: block;
+                margin-bottom: 10px;
+            }
+
+            .form-footer {
+                display: flex;
+                flex-direction: column;
+                button:last-child {
+                    margin-bottom: 10px;
+                    order: -1;
+                }
+                @media #{$mq-tablet-and-up} {
+                    flex-direction: row;
+                    justify-content: flex-end;
+                    button:last-child {
```

Comment:
Nit: Not sure if prettier did not kick in but, there should be an empty line above this one.